src/app/baseApi.ts

```diff
@@ -6,7 +6,7 @@ const getCsrfToken = () => getCookie('csrftoken');
 export const baseApi = createApi({
     baseQuery: fetchBaseQuery({
             baseUrl: import.meta.env.VITE_KF_API_URL,
-            credentials: "include",
+            // credentials: "include",
```

Comment:
Did you mean to commit this?

Reply:
Yes, this caused an authentication issue, so I’ve commented it out for future reference.